src/reducers/ModulesReducer.js

```diff
@@ -24,21 +24,20 @@ import { SYNC_TRANSACTION_COMPLETE } from '../sync/constants';
  * }
  */
 
-const checkModule = key => UIDatabase.getSetting(key).toLowerCase() === 'true';
-
 const initialState = () => {
   const usingInsurance = UIDatabase.objects('InsuranceProvider').length > 0;
   const usingPaymentTypes = UIDatabase.objects('PaymentType').length > 0;
   const usingPrescriptionCategories = UIDatabase.objects('PrescriptionCategory').length > 0;
   const usingSupplierCreditCategories = UIDatabase.objects('SupplierCreditCategory').length > 0;
 
-  const usingDashboard = checkModule(SETTINGS_KEYS.DASHBOARD_MODULE);
-  const usingDispensary = checkModule(SETTINGS_KEYS.DISPENSARY_MODULE);
-  const usingVaccines = checkModule(SETTINGS_KEYS.VACCINE_MODULE);
-  const usingCashRegister = checkModule(SETTINGS_KEYS.CASH_REGISTER_MODULE) && usingPaymentTypes;
-  const usingPayments = checkModule(SETTINGS_KEYS.PAYMENT_MODULE);
-  const usingPatientTypes = checkModule(SETTINGS_KEYS.PATIENT_TYPES);
-  const usingHideSnapshotColumn = checkModule(SETTINGS_KEYS.HIDE_SNAPSHOT_COLUMN);
+  const usingDashboard = UIDatabase.getPreference(PREF_KEYS.DASHBOARD_MODULE);
```

Comment:
could be pedantic and explicitly type each call for the `using..` variables as you expect them to be bools rather than null (super pedantic)

I couldnt find a gif about being pedantic so you get this

![chef](https://user-images.githubusercontent.com/35858975/77354508-d17fc880-6da7-11ea-8e09-f8f8fdc5b771.gif)

Reply:
9eb4b118cbfa521678f4c0b701a38c63a07a6a7a :+1: .